bat-utils/lib/runtime-wallet.js

```diff
@@ -359,7 +359,8 @@ Wallet.providers.uphold = {
       uphold.storage.setItem('uphold.access_token', info.parameters.access_token)
 
       user = await uphold.api('/me')
-      cards = await uphold.api('/me/cards')
+      authorized = user.status !== 'pending'
```

Comment:
this is less restrictive than the check it's replacing. I assume this is correct since you talked with uphold, however I can't check myself since their documentation doesn't explain possible values of this field as far as I can tell

Reply:
yes, they said that the test was to make sure the status was *not* `"pending"` ... hence this change.  and yes, the SDK doc could be a little clearer.